lib/iris/quickplot.py

```diff
@@ -117,10 +108,8 @@ def _label_with_points(cube, result=None, ndims=2, coords=None, axes=None):
 def _get_titles(u_object, v_object):
     if u_object is None:
         u_object = iplt._u_object_from_v_object(v_object)
-    xunits = u_object is not None and not u_object.units.is_time_reference()
```

Comment:
Are we losing anything without `u_object is not None`, or was that just 'asking permission' before running the rest of the line?